src/Microsoft.Health.Fhir.TemplateManagement/Utilities/StreamUtility.cs

```diff
@@ -84,5 +106,24 @@ public static string CalculateDigestFromSha256(Stream stream)
             hashedValue += string.Join(string.Empty, hashedStrings);
             return hashedValue;
         }
+
+        private static void FixTimestampInGzHeader(MemoryStream inputStream)
+        {
+            int num = (int)((GzipTimestamp.Ticks - new DateTime(1970, 1, 1).Ticks) / 10000000);
+            byte[] obj = new byte[4]
+            {
+                    0,
+                    0,
+                    0,
+                    0,
+            };
+            obj[0] = (byte)num;
+            obj[1] = (byte)(num >> 8);
+            obj[2] = (byte)(num >> 16);
+            obj[3] = (byte)(num >> 24);
+            byte[] array = obj;
+            inputStream.Seek(4, SeekOrigin.Begin);
+            inputStream.Write(array, 0, array.Length);
```

Comment:
need restore the position of the given stream

Reply:
Thanks, fixed it.